### PR TITLE
Feathr web ui prototype

### PR DESCRIPTION
**Pull request by blrchen**
This PR initialize feathr web portal prototype to allow user to create and manage features across projects.

- UX framework: site global style, layout, menu, header, etc.
- Azure AD login
- UI pages for feature creating/listing/editing and data source listing
- Add placeholders for following
  - User profile
  - Feature search
  - Project selection
  - Jobs
  
This PR uses a mock api server for feature CRUD.  The API work is still in progress, please expect follow up PRs with feature model changes to align the real API design and implementation.

On a site note, continues deployment to staging environment has been turned on. The url for web ui staging environment can be found in each PR's comment by github-actions.

### CODE REVIEW

**Review by xiaoyongzhu — APPROVED** [review]

File: ui/yarn.lock [hunk 1]
```diff
@@ -0,0 +1,9408 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+
```
Comment (hangfei) [inline ui/yarn.lock:4]: not familiar with this. Do we need to check in this?

**Review by hangfei — COMMENTED** [review]
Could you start to buidl a simple dev guide now? Just some headers are enough.

You can gradually adding things over time.

File: ui/yarn.lock (hunk 1 above)
Comment (blrchen) [reply ui/yarn.lock:4]: README has been updated to add details for Build and run locally, Deployment and Project Structure.

**Review by jaymo001 — APPROVED** [review]

**Comment by blrchen** [comment]
Already merged with a different PR, closing this original PR

**Pull request closed without merging**